src/go/configgenerator/filterconfig/filter_generator.go

```diff
@@ -271,6 +271,7 @@ func makeTranscoderFilter(serviceInfo *ci.ServiceInfo) (*hcmpb.HttpFilter, error
 					AlwaysPrintEnumsAsInts:     serviceInfo.Options.TranscodingAlwaysPrintEnumsAsInts,
 					PreserveProtoFieldNames:    serviceInfo.Options.TranscodingPreserveProtoFieldNames,
 				},
+				MatchUnregisteredCustomVerb: serviceInfo.Options.TranscodingMatchUnregisteredCustomVerb,
```

Comment:
The correct behaviors should be "on",  is that correct?
Should we just turn it on by default?  
is this a breaking change?

Reply:
+1, how likely is it to break users? Silently turning it on seems fine to me

Reply:
Yes, it could a breaking change so I prefer to disable it. Internally, we can enable it by default.

As it is the issue effecting internal users, we just don't  expose to ESPv2 customers for now(most likely, ESPv2 users don't use custom verbs).

Reply:
Ok, keep it disabled externally and enable internally SG